Content.Server/WebPlacer/WebPlacerSystem.cs

```diff
@@ -0,0 +1,100 @@
+using System.Linq;
+using Content.Server.Popups;
+using Content.Shared.Maps;
+using Content.Shared.WebPlacer;
+using Content.Shared.Whitelist;
+using Robust.Server.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Map;
+using Robust.Shared.Map.Components;
+
+namespace Content.Server.WebPlacer;
+
+/// <summary>
+/// Spawns entities (probably webs) around the component owner. Action handled by <see cref="SharedWebPlacerSystem"/>.
+/// </summary>
+public sealed class WebPlacerSystem : SharedWebPlacerSystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly EntityWhitelistSystem _whitelistSystem = default!;
```

Comment:
```suggestion
    [Dependency] private readonly EntityWhitelistSystem _whitelist = default!;
```
codestyle convention